packages/solidity-contracts/scripts/hardhat/verifyDeployment.ts

```diff
@@ -0,0 +1,123 @@
+import { task } from 'hardhat/config';
+import { HardhatRuntimeEnvironment } from 'hardhat/types';
+import { config as dotEnvConfig } from 'dotenv';
+import { ContractFactory } from 'ethers';
+import { writeFileSync } from 'fs';
+
+dotEnvConfig();
```

Comment:
```suggestion
import { ContractFactory } from 'ethers';
import { writeFileSync } from 'fs';
```

Hardhat already does dotenv configuration, is it needed here?

Reply:
yeah removed